src/index.html

```diff
@@ -17,7 +17,42 @@
     <body>
       <section class="container">
 
-      <section class="small-rect"></section>
+      <section class="small-rect">
+        <section class="content">
+          <section id="pict">
```

Comment:
lots of `<sections>` here! Check out other semantic HTML you can use: https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Sections_and_Outlines_of_an_HTML5_document